src/window/ui.xml

```diff
@@ -0,0 +1,200 @@
+                <property name="digits">4</property>
+                <property name="value-pos">left</property>
+                <layout>
+                  <property name="column">3</property>
+                  <property name="row">0</property>
+                </layout>
+              </object>
+            </child>
+            <child>
+              <object class="GtkScale" id="sdy_scale">
+                <property name="draw-value">True</property>
+                <property name="adjustment">sdy</property>
+                <property name="round-digits">1</property>
+                <property name="digits">4</property>
+                <property name="value-pos">left</property>
+                <layout>
+                  <property name="column">3</property>
+                  <property name="row">1</property>
+                </layout>
+              </object>
+            </child>
+            <child>
+              <object class="GtkLabel">
+                <property name="label" translatable="1">Pitch</property>
+                <layout>
+                  <property name="column">0</property>
+                  <property name="row">2</property>
+                </layout>
+              </object>
+            </child>
+            <child>
+              <object class="GtkLabel">
+                <property name="label" translatable="1">Roll</property>
+                <property name="use-underline">1</property>
+                <layout>
+                  <property name="column">2</property>
+                  <property name="row">2</property>
+                </layout>
+              </object>
+            </child>
+            <child>
+              <object class="GtkScale" id="pitch_scale">
+                <property name="draw-value">True</property>
+                <property name="adjustment">pitch</property>
+                <property name="round-digits">1</property>
+                <property name="digits">4</property>
+                <property name="value-pos">left</property>
+                <layout>
+                  <property name="column">1</property>
+                  <property name="row">2</property>
+                </layout>
+              </object>
+            </child>
+            <child>
+              <object class="GtkScale" id="yaw_scale">
+                <property name="draw-value">True</property>
+                <property name="adjustment">yaw</property>
+                <property name="round-digits">1</property>
+                <property name="digits">4</property>
+                <property name="value-pos">left</property>
+                <layout>
+                  <property name="column">3</property>
+                  <property name="row">2</property>
+                </layout>
+              </object>
+            </child>
+          </object>
+        </child>
+        <child>
+          <object class="GaussianPlot" id="plot">
+            <property name="vexpand">True</property>
```

Comment:
```suggestion
            <property name="vexpand">True</property>
```
this should probably be set from the widget itself on constructed or not?

Reply:
Done via the default window size instead, that makes more sense